Messaging.md

```diff
@@ -93,13 +93,14 @@ This type contains information about a contracts outcomes and their correspondin
 
 1. type: 42768 (`contract_info_v0`)
 2. data:
+   * [`u16`:`nb_outcomes`]
```

Comment:
UInt16 is probably not enough for a very large contract, worth using UInt32

Reply:
updated to use `bigsize` (a bit more efficient if number of outcome is small)